### DISCUSSION TITLE
v8.1.82 version not opening on some language settings (fixed in v8.1.83)

### DISCUSSION THREAD

**Discussion by pooni21** [tribalfs/GalaxyMaxHzPub#228] [Q&A] [answered]
I'm facing an issue with the latest version (v8.1.82) of the app on my Galaxy Z Fold 3 running One UI 6.1. While the previous version (1.8.7x) worked fine, the new one is giving me a version incompatibility error. I installed it using ADB. Could you please provide a solution or an estimated timeline for a fix?

**Comment by tribalfs** [comment]
Hi!

Can you try clearing the app's data if this resolves the issue?

If it's still crashing, would you mind providing us the crash log that we can look into?
To do so, you have to connect your phone to ADB and do the following:
1. Execute
`adb logcat -c`
2. Run the app and let it crash
3. Execute
`adb logcat -d -b crash > gmh_crashlog.txt`
4. Locate gmh_crashlog.txt file inside adb folder and send it to us.

Thank you.

**Reply by pooni21** [reply]
[gmh_crashlog.txt](https://github.com/user-attachments/files/16555880/gmh_crashlog.txt)
I have attached the log file for your reference.

Thank you.

**Reply by tribalfs** [reply, marked as answer]
Thank you.

Please try this [build](https://github.com/tribalfs/GalaxyMaxHzPub/releases/download/v8.1/galaxy_max_hz_v8.1.apk).

**Reply by pooni21** [reply]
It was resolved well. 
thank you
